### ISSUE TITLE
JTR Router misinterprets "via" argument

### ISSUE THREAD

**Issue by leoluecken** [DLR-TS/sumo#3566]
When using JTR router, not specifying sinks, but giving "via" attributes in the flow definitions (no "to" attributes), vehicles are always routed from the "from"-edge to the "via"-edge.
(The via edges are located prior to any junction by the way).

**Comment by namdre** [comment]
I don't think it makes sense for jttrouter to use the 'via' attribute at all. (jtrrouter also does not make use of the 'to' attribute. It would probably be a good idea to issue a warning if either of those attributes are present.
